### ISSUE TITLE
How to install?

### ISSUE THREAD

**Issue by Mostafa20946** [uikit/uikit-site#104]
How to install??

**Comment by Alymbek** [comment]
1. install `npm`
2. install `bower`
3. create separate `locahost` (optional, but I recommend)
4. set `full permission` for folder of localhost (mandatory)
5. clone the repository `https://github.com/uikit/uikit-site.git` to folder of localhost **OR**  download source files by running `npm install uikit-site` (in this case move uikit content to core folder of localhost)
7. install by `npm install`
8. run `npm run setup` to compile
9. check in browser
10. download external javascript library from links in **index.html** and **docs/index.html** to make it local 
11. edit javascript  links in **index.html** and **docs/index.html** from external to internal files
12. check in browser again


ps. istall `npm` and `bower` with `sudo` privilege.

pps. `npm` command running in CLI.

ppps. you'll get error message while installing if doing wrong, just try to fix it.

pppps. to fully make it offline ensure to download external javascript files and make it locally:
for  **index.html** file : 
        <script src="https://www.google-analytics.com/analytics.js" async></script>
        <script src="https://cdnjs.cloudflare.com/ajax/libs/autotrack/2.3.2/autotrack.js" async></script>
        <script src="https://cdnjs.cloudflare.com/ajax/libs/marked/0.3.6/marked.min.js"></script>
        <script src="https://cdnjs.cloudflare.com/ajax/libs/vue/2.2.6/vue.min.js"></script>
        <script src="https://cdnjs.cloudflare.com/ajax/libs/vue-router/2.5.1/vue-router.min.js"></script>

for **docs/index.html** file : 
         <script src="https://www.google-analytics.com/analytics.js" async></script>
        <script src="https://cdnjs.cloudflare.com/ajax/libs/autotrack/2.3.2/autotrack.js" async></script>
        <script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/1.6.1/clipboard.min.js"></script>
        <script src="https://cdnjs.cloudflare.com/ajax/libs/he/1.1.1/he.min.js"></script>
        <script src="https://cdnjs.cloudflare.com/ajax/libs/marked/0.3.6/marked.min.js"></script>
        <script src="https://cdnjs.cloudflare.com/ajax/libs/vue/2.2.6/vue.min.js"></script>
        <script src="https://cdnjs.cloudflare.com/ajax/libs/vue-router/2.5.1/vue-router.min.js"></script>

**Comment by Alymbek** [comment]
@Mostafa20946, please, don't close this issue for info for other beginners.
I just try to make page for wiki of this project, but wiki is disabled.

**Comment by Mostafa20946** [comment]
@Alymbek , Thanks for the tutorial.
Is it possible to install on Cpanel or Directadmin ?

**Comment by aarongerig** [comment]
@Mostafa20946 Yup, that‘s possible, but not exactly the right place to ask such question. This doesn‘t have anything to do with UIkit.

@Alymbek The tutorial actually is right here: https://github.com/uikit/uikit-site/blob/develop/README.md
Your example is one way to do it, but actually you won‘t need a localhost necessarily. Just my two cents... 😉

**Comment by Alymbek** [comment]
@aarongerig , this steps is for total beginners.

in that manual no info about folder permission, see [https://github.com/uikit/uikit-site/issues/100#issuecomment-353960209](https://github.com/uikit/uikit-site/issues/100#issuecomment-353960209).

I marked localhost as optional, you are right, it's not necessarily

**Comment by aarongerig** [comment]
@Alymbek true, why don‘t you submit a PR to this repo, so beginners see it right away?

**Comment by Alymbek** [comment]
I don't know how to do it, I'm also beginner in github...

I think the best solution is enable wiki and make page in wiki

**Comment by steven-ferrer** [comment]
Hmmm. I think this should be on the README

**Comment by Alymbek** [comment]
Hi, friends.
There new process to install locally last version, please check here #152 

You can close this issue, because out of date.

**Comment by ViktorJays** [comment]
not working for me. pls help
